build/README.md

```diff
@@ -1,60 +1,170 @@
+# Start with docker compose
+
+You can start ZITADEL with a simple docker compose up.
+
+The services are configured to restart if an error occurs. Until the essential services are started and executed some services panic, this is expected just give it some minutes to setup the database and execute migrations.
+
+```bash
+COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker compose -f ./build/local/docker-compose-dev.yml --profile init --profile backend --profile frontend -p zitadel up
+```
```

Comment:
Can we show a success message, so that the user knows when it's done?

Reply:
We could run a separate script until readiness returns OK. I'll create a script tomorrow.